exercises/practice/phone-number/.docs/instructions.append.md

```diff
@@ -1,22 +1,20 @@
 # Instructions append
 
-Since this exercise has difficulty 5 it doesn't come with any starter implementation.
-This is so that you get to practice creating classes and methods which is an important part of programming in Java.
-It does mean that when you first try to run the tests, they won't compile.
-They will give you an error similar to:
+This exercise doesn't come with any starter implementation.  This is so that you get to practice creating classes and
+methods, which is an important part of programming in Java.  That means that when you first try to run the tests, they
+won't compile.  They will give you an error similar to:
```

Comment:
We adhere to the [Markdown style](https://exercism.org/docs/building/markdown/markdown) given by Exercism, which is one sentence per line. Can you revert these changes in this file?